dotnet/src/SemanticKernel.Abstractions/Schema/JsonSchemaFunctionView.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.SemanticKernel;
 /// <summary>
 /// A class to describe an SKFunction in a Json Schema friendly way.
 /// </summary>
-public sealed class JsonSchemaFunctionManual
+public sealed class JsonSchemaFunctionView
 {
     /// <summary>
     /// he function name.
```

Comment:
While you're editing, can you fix this?
```suggestion
    /// The function name.
```